src/render/view/components/Controls.tsx

```diff
@@ -101,6 +153,16 @@ export class Controls extends React.Component<IControlsProps, IControlsState> {
         }
     }
 
+    protected componentWillUpdate(nextProps: IControlsProps, nextState: IControlsState) {
+        // Calucalte value for volume slider
+        if (nextState.mute) {
```

Comment:
Still need to fix the volume/mute defect that was solved in Right2Drive/ease-web